ae/tests/deployment_test.yaml

```diff
@@ -251,3 +251,28 @@ tests:
       - equal:
           path: spec.template.spec.serviceAccountName
           value: "test-sa"
+  - it: Deploy without custom SecurityContext
+    template: engine-deployment.yaml
+    set:
+      engine:
+        runAsNonRoot: false
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Deployment
+      - isNull:
+          path: spec.template.spec.containers[0].securityContext
+  - it: Deploy with custom  SecurityContext
+    template: engine-deployment.yaml
+    set:
+      engine:
+        runAsNonRoot: true
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Deployment
+      - equal:
+          path: spec.template.spec.containers[0].securityContext.runAsNonRoot
+          value: true
```

Comment:
@gigo1980 could you also check the default `spec.template.spec.containers[0].securityContext.runAsUser` 

Thank you !

Reply:
done